src/php/views/partials/content-none.twig

```diff
@@ -1,9 +1,3 @@
-<section>
-	<header>
-		<h1>Nothing Found</h1>
-	</header>
-
-	<div>
-		{{ function('get_search_form') }}
-	</div>
-</section>
+<div class="util-full-width">
```

Comment:
Instead of having a class for `util-full-width`, it might make more sense to apply `margin-inline-start: auto` to `.cmp-primary-sidebar`. This should push it to the right side of the page without requiring any interventions in the main content area.

Reply:
🤯 The property I never knew I needed. Thanks!